apm-lambda-extension/extension/process_env.go

```diff
@@ -25,12 +25,13 @@ import (
 )
```

Comment:
Addition of the timeout env variable. It is in uppercase in the `extensionConfig` struct because it needs to be exported. In the end, I chose to keep this change into the PR, as it is required for all the Unit Tests to pass.